### PR TITLE
READMES upload

### PR DESCRIPTION
**Pull request by Enzito-dev**
**Description**
New sub-directories added in the repository, each folder created contains its own readme.
The first README contains a link to both sub-directories.

**Changes made**

- README.md updated
- Java (folder created)
        - README.md created
- JavaScript (folder created)
        - README.md created

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,12 @@
+# CodeRepo
 
+Collaborators:
+
++ Hernan Cano
```
Comment (emebustoss) [inline README.md:5]:
```suggestion
+ Enzo Avena
+ Gonzalo Muñoz
+ Hernan Cano
+ Maximiliano Bustos
```

I think that the README file would look better if the names are alphabetically ordered.
Or with Surnames.

+ Avena Enzo
+ Bustos Maximiliano
+ Cano Hernan
+ Muñoz Gonzalo

File: README.md [hunk 2]
```diff
@@ -1 +1,12 @@
...
++ Enzo Avena
```
Comment (totigm) [inline README.md:9]:
```suggestion
+ Gonzalo Muñoz
```

File: README.md [hunk 3]
```diff
@@ -1 +1,12 @@
...
++ Maximiliano Bustos
```
Comment (totigm) [inline README.md:10]:
```suggestion
+ Hernán Cano
```

File: README.md [hunk 4]
```diff
@@ -1 +1,12 @@
...
++ Gonzalo Muñoz
```
Comment (totigm) [inline README.md:11]:
```suggestion
+ Maximiliano Bustos
```

File: README.md [hunk 5]
```diff
@@ -2,11 +2,14 @@
...
 Collaborators:
 
-+ Hernan Cano
+Enzo Avena
+Gonzalo Muñoz
+Hernan Cano
+Maximiliano Bustos
 + Enzo Avena
 + Maximiliano Bustos
 + Gonzalo Muñoz
```
Comment (Hernancano98) [inline README.md]: **Changes:**
The changes commited were wrong written, i'm correcting theme.

```suggestion
+ Hernan Cano
+ Enzo Avena
+ Maximiliano Bustos
+ Gonzalo Muñoz
```

File: JavaScript/README.md [hunk 6]
```diff
@@ -0,0 +1,6 @@
+# JavaScript
+
+## Sub-directories:
+
+
+[Back to root](../README.md)
```
Comment (totigm) [inline JavaScript/README.md:6]:
```suggestion
[<- Back to root](../README.md)
```

File: Java/README.md [hunk 7]
```diff
@@ -0,0 +1,6 @@
+# Java
+
+## Sub-directories:
+
+
+[Back to root](../README.md)
```
Comment (totigm) [inline Java/README.md:6]:
```suggestion
[<- Back to root](../README.md)
```

File: README.md [hunk 8]
```diff
@@ -1 +1,12 @@
+# CodeRepo
 
+Collaborators:
+
++ Hernan Cano
++ Enzo Avena
++ Maximiliano Bustos
++ Gonzalo Muñoz
+
+[Java](./Java/README.md)
```
Comment (totigm) [inline README.md:13]:
```suggestion
```

File: README.md [hunk 9]
```diff
@@ -1 +1,12 @@
...
+
+[JavaScript](./JavaScript/README.md)
```
Comment (totigm) [inline README.md]:
```suggestion
```

File: README.md [hunk 10]
```diff
@@ -1 +1,12 @@
+# CodeRepo
```
Comment (totigm) [inline README.md:1]:
```suggestion
# Project-Book

In this repo you'll find different projects in different languages, such as:

+ [Java](./Java/README.md)
+ [JavaScript](./JavaScript/README.md)
```

File: README.md [hunk 11]
```diff
@@ -1 +1,19 @@
+# Project-Book
 
+In this repo you'll find different projects in different languages, such as:
+
++ [Java](./Java/README.md)
++ [JavaScript](./JavaScript/README.md)
+
+Collaborators:
+
+Enzo Avena
```
Comment (totigm) [inline README.md:10]:
```suggestion
```

File: README.md [hunk 12]
```diff
@@ -1 +1,19 @@
...
+Gonzalo Muñoz
```
Comment (totigm) [inline README.md:11]:
```suggestion
```

File: README.md [hunk 13]
```diff
@@ -1 +1,19 @@
...
+Hernan Cano
```
Comment (totigm) [inline README.md:12]:
```suggestion
```

File: README.md [hunk 14]
```diff
@@ -1 +1,19 @@
...
+Maximiliano Bustos
```
Comment (totigm) [inline README.md:13]:
```suggestion
```

File: README.md [hunk 15]
```diff
@@ -1 +1,19 @@
...
++ Gonzalo Muñoz
++ Hernán Cano
++ Maximiliano Bustos
+
+
+[JavaScript](./JavaScript/README.md)
```
Comment (totigm) [inline README.md:19]:
```suggestion
```

**Pull request closed without merging**